pkg/az/az.go

```diff
@@ -12,7 +12,7 @@ import (
 
 const (
 	// DefaultRequiredAvailabilityZones defines the number of required availability zones
-	DefaultRequiredAvailabilityZones = 3
+	DefaultRequiredAvailabilityZones = 2
```

Comment:
As I mentioned in #290, it would be best to actually keep default to 3 here, but allow 2 in certain cases (namely the `us-east-1` region). So I'd change this to:

```suggestion
	DefaultRequiredAvailabilityZones = 3
	MinRequiredAvailabilityZones = 2
```

Reply:
For background on us-east-1, see #118 and #252.